docs/articles/monetization-limitations.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Limitations
+---
+
+Zuplo strives to deliver the most complete, flexible, and intuitive API
+monetization system. Given the multitude of different pricing models and their
+various configurations - we unfortunately have limited support for certain
+features you may want for your monetization model. If the current offering does
+not currently suit your needs, please
+[reach out](mailto:support@zuplo.com?subject=Monetization%20Inquiry) and we can
+expedite work on it.
```

Comment:
```suggestion
monetization system. Given the multitude of different pricing models and their
various configurations, if the current offering does
not currently suit your needs, please
[reach out](mailto:support@zuplo.com?subject=Monetization%20Inquiry) and we can
expedite work on it.
```

Reply:
I would change it to something like this

Reply:
Whoops merged already. Will apply in another commit